ui/design-system-core/src/components/DuplexButton/DuplexButton.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import DuplexButton from './';
+
+export default {
+  title: 'DuplexButton',
+  component: DuplexButton,
+};
+
+const Template = args => {
+  const [active, setActive] = React.useState(false);
+
+  const handleClick = () => {
+    if (!active) {
+      setActive(true);
```

Comment:
this if else can be simplified as `setActive(!active)`

Reply:
:) Thanks for pointing it out Marius.